src/nupic/algorithms/Cells4.hpp

```diff
@@ -406,8 +407,8 @@ namespace nupic {
                Real permInc =.1,
                Real globalDecay =0,
                bool doPooling =false,
-               int seed =-1,
-               bool doItAll =false,
+               int seed =42,
+               bool initFromCpp =true,
```

Comment:
I like this name change. I am confused about the default of `true` here when it was previously `false`. The default for this parameter later in the file was left as `false` so I think I would keep it `false` here as well.